
Fixed dvdauthor for gcc >= 3.4.x - the usual suspects.

  - Rene Rebe <rene@exactcode.de>

--- ./src/dvdvob.c.vanilla	2004-11-05 12:44:29.369409772 +0100
+++ ./src/dvdvob.c	2004-11-05 12:44:36.417142892 +0100
@@ -1149,6 +1149,7 @@
                         fprintf(stderr,"\n");
                         ach->audpts[ach->numaudpts-1].pts[1]=pts0;
                     noshow:
+			;
                     }
                     ach->audpts[ach->numaudpts].pts[0]=pts0;
                     ach->audpts[ach->numaudpts].pts[1]=pts1;
